### ISSUE TITLE
Response delayed when accessing HTTPs website 

### ISSUE THREAD

**Issue by sealboy100** [jiejieTop/http-client#7]
I encountered a problem when using the http-client to access HTTPs website.
It takes about 20 seconds to get response from HTTPs website and the response time is too long.
Does anybody have the same problem when accessing HTTPs website?
What is the possible root cause?
Hope someone can answer for me, thanks.

**Comment by tccxy** [comment]
yes ,i get the same problem ,the time of https web access is too long....about 20s...
Is there any way to solve it??

**Comment by tccxy** [comment]
i change the "LOG_LEVEL" from INFO_LEVEL ->WARN_LEVEL and modiphy the "HTTP_DEFAULT_CMD_TIMEOUT" from 20000 to 3000 ,it's semms to be working, the time of access HTTPs web about 3second??

the change file is "http_config.h"
